src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -112,31 +154,36 @@ Version VSVersion
             }
         }
 
-        protected override async tasks.Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
+        protected override Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
             AddService(typeof(IUIProvider), CreateService, true);
-
-            // Load the start page package only for Dev15 Preview 4
-            if (VSVersion.Major == 15 && VSVersion.Build == 25618)
-            {
-                var shell = await GetServiceAsync(typeof(SVsShell)) as IVsShell;
-                IVsPackage startPagePackage;
-                if (ErrorHandler.Failed(shell?.LoadPackage(new Guid(StartPagePreview4PackageId), out startPagePackage) ?? -1))
-                {
-                    // ¯\_(ツ)_/¯
-                }
-            }
+            AddService(typeof(IUsageTracker), CreateService, true);
+            AddService(typeof(IMenuProvider), CreateService, true);
+            return Task.FromResult<object>(null);
```

Comment:
I feel ``Task.FromResult(true)`` is slightly easier to read than  `Task.FromResult<object>(null)`. Less brackets to track while mentally parsing.